MAUI/Scheduler/drag-and-drop-appointments.md

```diff
@@ -0,0 +1,245 @@
+{
+    var appointment = e.Appointment;
+    var draggingPoint = e.DragPoint;
+    var draggingTime = e.DragTime;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Handle appointment drop
+Using the `AppointmentDrop` event you can get the dropping appointment details, position, time and you can handle whether the appointment can be dropped to the specific position or not. This event will trigger after dropping the appointment. The `AppointmentDropEventArgs` argument contains the following properties.
+
+`Appointment` - Get the appointment that is being dragged and dropped.
+`DropTime`- Get or set the date and time at which the appointment is being dropped.
+`SourceResource` - Get the original resource of the appointment that is being dropped.
+`TargetResource` - Get the resource into which the appointment is being dropped.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDrop += OnSchedulerAppointmentDrop;
+
+...
+
+private void OnSchedulerAppointmentDrop(object? sender, AppointmentDropEventArgs e)
+{
+        var appointment = e.Appointment;
+        e.Cancel = false;
+        var dropTime = e.DropTime;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Change the drag and drop settings
+You can change the appointment of drag and drop settings in Scheduler by using the `DragDropSettings` property.
+
+{% tabs %}
+{% highlight xaml %}
+<schedule:SfScheduler x:Name="scheduler" ShowBusyIndicator="True"
+                    AppointmentsSource="{Binding DragEvents}"
+                    View="Week"
+                    DisplayDate="{Binding DisplayDate}"
+                    AllowedViews="Day,Week,WorkWeek,Month">
+        <schedule:SfScheduler.DragDropSettings>
+            <schedule:DragDropSettings AllowNavigation="true" AllowScroll="true" 
+                                    ShowTimeIndicator="true"/>
+        </schedule:SfScheduler.DragDropSettings>
+</schedule:SfScheduler>
+{% endhighlight %}
+{% highlight c# %}
+scheduler.DragDropSettings = new DragDropSettings 
+{
+    AllowNavigation = true,
+    AllowScroll = true,
+    ShowTimeIndicator = true,
+};
+{% endhighlight %}
+{% endtabs %}
+
+#### Disable view navigation when dragging appointment
+Using the `AllowNavigate` property of the `DragDropSettings` you can handle whether you can navigate views while dragging the appointment or not. The default value of the AllowNavigate property is `true`.
+
+{% tabs %}
+{% highlight xaml %}
+<schedule:SfScheduler x:Name="Scheduler"
+                    View="Week">
+<schedule:SfScheduler.DragDropSettings>
+    <schedule:DragDropSettings AllowNavigation="False">
+    </schedule:DragDropSettings>
+</schedule:SfScheduler.DragDropSettings>
+</schedule:SfScheduler>
+{% endhighlight %}
+{% highlight c# %}
+this.Scheduler.View = SchedulerView.Week;
+this.Scheduler.DragDropSettings.AllowNavigation = false;
+{% endhighlight %}
+{% endtabs %}
+
+#### Handle navigation delay for dragged appointment
```

Comment:
**dragged** appointment - the term, right? And did u get a content review?

Reply:
Changed **Handle dragging appointment navigation delay**. Yes, Got content review.